ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -122,7 +124,8 @@ import { AbstractActivityQuestionBlock } from '../../models/activity/abstractAct
                                             [studyGuid]="studyGuid"
                                             [activityGuid]="activityGuid"
                                             (visibilityChanged)="updateVisibility($event)"
-                                            (embeddedComponentValidStatusChanged)="updateEmbeddedComponentValidationStatus(2, $event)"
+                                            (addressComponentValidStatusChanged)="updateEmbeddedComponentValidationStatus(4, $event)"
+                                            (institutionComponentValidationStatusChanged)="updateEmbeddedComponentValidationStatus(5, $event)"
                                             (embeddedComponentBusy)="embeddedComponentBusy$[2].next($event)">
```

Comment:
See my comment below about consolidating the two "*ComponentValidationSttusChanged" variable. Notice that you treat them here (correctly) as being two objects of the same thing.